seed/views/v3/organizations.py

```diff
@@ -1392,3 +1403,353 @@ def reset_all_passwords(self, request, pk=None):
                 'message': 'passwords reset'
             }
         )
+
+    @has_perm_class('requires_member')
+    @ajax_request_class
+    @action(detail=True, methods=['GET'])
+    def insert_sample_data(self, request, pk=None):
+        """
+        Create a button for new users to import data below if no data exists
+        """
+        org = Organization.objects.get(id=pk)
+        cycles = Cycle.objects.filter(organization=org)
+        if cycles.count() == 0:
+            return JsonResponse({
+                'status': 'error',
+                'message': 'there must be at least 1 cycle'
+            }, status=status.HTTP_400_BAD_REQUEST)
+        cycle = cycles.first()
```

Comment:
I'm wondering if we should also add a check like we do on FE to make sure this cycle doesn't have any inventory?